core/src/main/java/io/aiven/klaw/helpers/db/rdbms/InsertDataJdbc.java

```diff
@@ -162,30 +162,26 @@ public synchronized Map<String, String> insertIntoRequestConnector(
     return hashMap;
   }
 
-  public synchronized String insertIntoTopicSOT(List<Topic> topics) {
-    Set<Topic> existingTopicIds = new HashSet<>();
+  public synchronized CRUDResponse<Topic> insertIntoTopicSOT(List<Topic> topics) {
+    Set<Integer> existingTopicIds = new HashSet<>();
```

Comment:
I think this variable could be renamed to nonExistingTopics

Reply:
So this is a list of existing topic Ids though, the query gets a list of all topic Ids that already exist within the database that are also in the the variable "topics".